exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/SQLExpressionBuilder.kt

```diff
@@ -96,6 +93,37 @@ fun <T : Any?> ExpressionWithColumnType<T>.varPop(scale: Int = 2): VarPop<T> = V
  */
 fun <T : Any?> ExpressionWithColumnType<T>.varSamp(scale: Int = 2): VarSamp<T> = VarSamp(this, scale)
 
+// Array Comparisons
+
+/** Returns this subquery wrapped in the `ANY` operator. This function is not supported by the SQLite dialect. */
+fun <T> anyFrom(subQuery: Query): Op<T> = AllAnyFromSubQueryOp(true, subQuery)
```

Comment:
The `ANY` and `ALL` operators with subquery arguments are not supported by SQLite as is also commented in the commit message. Actually `ANY` and `ALL` are not keywords of SQLite at all according to [this SO question](https://stackoverflow.com/questions/35333660/sqlite-syntax-for-operator-any).

Reply:
@ShreckYe I forgot to mention something. Please take `AbstractQuery<*>` as the argument so we don't restrict it to just `Query`. It should be possible to use a `SetOperation` for example as a subquery in the SQL.